src/api/payment-requests/payment-requests.md

```diff
@@ -224,6 +226,18 @@ Payment Activities are created when a Payment Request has been **created**, **pa
 | cancellation | [Payment Request][] was cancelled by the merchant or the shopper.  |
 | expiry       | [Payment Request][] wasn't paid before time out.                   |
 
+<a name="cancellation-reasons">
+{% h4 Cancellation Reasons %}
```

Comment:
We originally agreed to have this as a heading inline with payment activities but it didnt really make sense because a cancellation reason is not a model. This table now exists in the Payment Activities model with a link to it from the payment request model too.